src/generalComponents/BlueButtons.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components"
+// import iconRight from "../image/iconRight.svg"
+
+const StyledBlueButtons = styled.button`
+background-color: ${(props)=> props.backgroundColor || '#1A56DB'};
```

Comment:
zdecydowanie należy uprościć style. Jeśli występuje kilka wariantów przycisku, można to ograć jednym propsem i w oparciu o jego wartość stylować warunkowo. Szerokość powinna być responsywna i ograniczona właściwościami max-width i min-width